vertx-web/src/test/java/io/vertx/ext/web/RouterTest.java

```diff
@@ -1995,4 +1996,20 @@ public void testGetWithPlusPath2() throws Exception {
     });
     testRequest(HttpMethod.GET, "/some+path?q1=some+query", 200, "foo");
   }
+
+  @Test
+  public void testCallOnce() throws Exception {
```

Comment:
it's not :-/ some test i wrote to validate some report, i can remove it or let one extra test in, any preference?